lib/engine/assignable.rb

```diff
@@ -17,5 +17,14 @@ def assign!(key, value = true)
     def remove_assignment!(key)
       assignments.delete(key)
     end
+
+    def self.remove_from_all!(assignables, key)
```

Comment:
should we just remove from the previous as opposed to doing everything?

Reply:
Looking at previous could be problematic. Since order could be:

1. Assign corp
2. Assign hex
3. Re-assign corp.

The previous in #3 would be the hex but we want to leave that and remove the corp.